### ISSUE TITLE
Possible changes to spawn selection feature

### ISSUE THREAD

**Issue by bchawke** [DayZMod/DayZ#744]
The current spawn selection feature has a certain failure rate programmed into it, but it isn't high enough and players can effectively spawn where they want a vast majority of the time on servers that have spawn selection.  In my opinion this ruins any type of squad combat near the coast because players can be back in the fight in under a minute after being killed.  Wiping a squad becomes nearly impossible and instead it's just a respawn-and-run-back-to-my-body fest until one side finally gives up.

This issue becomes amplified when base building is added.  1.8.6 proved that squads building a base close to a coastal spawn had far too much of an advantage over other players by being able to return to their base within one minute of dying, and accessing gear on their dead body that was protected by the base walls.

One approach to fix this would be to raise the failure rate so that it's a real roll of the dice as to whether or not you get your preferred spawn (if you took this route you would need to make this very clear to new players, possibly put text at the bottom of the spawn selection screen stating that the spawn choice is not guaranteed).

Another approach, which I think would be better, would be to reduce the spawn selection screen to simply two choices: South Coast and East Coast.  If you wanted a Cherno spawn, you'd choose South Coast, but it could put you anywhere from Kamenka to Kamyshovo.  If you wanted a Berezino spawn, you'd pick East Coast but be spawned anywhere from Krutoy Cap to Berezino.

**Comment by Inurosen** [comment]
Totally agree with your concerns.
I like the idea of having two choices.

**Comment by KPanicEKS** [comment]
I love the idea of 2 spawn selections as well. The only addition I would
like to see incorporated would be some sort of suicide detection system
that would prevent the players from instantly committing suicide in effort
to get that spawn they were hoping for. Maybe if they die within 10 minutes
from spawning they get the same spawn point. Only after that 10 minute
would they get the chance for a new dice roll.

**Comment by Inurosen** [comment]
@M0tzilla i don't think we should prevent people from suicide respawning on other points. If they want to die then let them die. I think it's okay as long it takes their time to get better spawn instead of just running through and possibly getting to desired point with fresh loot. Usually it takes almost same time.

**Comment by skigoggles** [comment]
just happened to be looking at this code - if I understand it correctly, there are 6 spawn spots and you will spawn within 1400 meters of one, but always on the coast.  thinking out loud - you could increase that distance, add a minimum distance and/or don't allow spawning within so many meters of the body (which 
I think some servers do that use the ESSSpawnSelector) . Catching suicide is hard, because dead is dead.

**Comment by AndrewEscocia** [comment]
I do understand the concerns raised by BC but I find the "60 seconds to be back in the fight" a little disingenuous. 

I think 60 secs is the best possible outcome and really not very likely.

For example, Lets say you are in a gun fight in east Cherno super market . You are pretty good and take down 4 or 5 of them . They are salty af cos they just got curbed and all spawn again in cherno . How many of the spawns do you think will be even on the east side of the city, maybe 3 ? of them maybe 1 gets  close enough to the market to return in 60 secs. 
Now I know that you are unlikely to kill all 5 in 1 go , so maybe by the time you have killed the last one, the first one is back thus dragging the fight on a little  but both sides have the same advantage.

Now lets take other spawn points . Soli can give spawn points from Kamy. Berez can give north Soli spawns. both way over a 60 sec jog.  

Lets look at taking away spawn select or making it a less affective time saver. 

The squad mate that dies in Berez gets spawned in Balota, will he make the trek back to Berez to meet his mates or just log out? 
Do we really want to make the running from zeds with only flares and a bandage a part of the game that people have to play again and again with out a concession to streamline it somewhat. 

Or if you are less fair minded you can game the system and suicide till they get the spawn they want.
